components/cam/src/physics/cam/micro_p3_utils.F90

```diff
@@ -268,14 +275,18 @@ subroutine micro_p3_utils_init(cpair,rair,rh2o,rhoh2o,mwh2o,mwdry,gravit,latvap,
     micro_liq_props = MicroHydrometeorProps(rhow, dsph, &
          min_mean_mass=min_mean_mass_liq)
   
+    ! AaronDonahue: Do we need any of these micro_XXX_props?  After switching to
+    ! evaporate_sublimate_precip and ice_depostion_sublimation are these used?
+    ! Even if they are used, should we do something else, maybe lookup table
+    ! based?
```

Comment:
microhydrometeorprops is just a derived datatype with info like density and lamda vals. It doesn't seem to be used anywhere (for ice, or for liq above here). If that's the case, we might as well delete it. I'm confused what it was originally used for in the version of P3 we got from WRF.

Reply:
This is actually needed for the interface routine we created.  And was brought over by copying the interface layer for MG.  My opinion is that we don't want to use it and need to use the lookup tables for consistentcy.  For reference, only ice and liq are actually used in the interface and they are used to determine `rel` and `rei` using the `size_dist_param_basic` subroutine that we decided we didn't want to use in determining qrevp, qidep, qisub and berg.  Once we replace these two calls I think we can remove all micro_XXX_props.